x-pack/test/plugin_api_integration/test_suites/event_log/public_api_integration.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { merge, omit, times, chunk } from 'lodash';
+import uuid from 'uuid';
+import expect from '@kbn/expect/expect.js';
+import moment from 'moment';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { IEvent } from '../../../../plugins/event_log/server';
+import { IValidatedEvent } from '../../../../plugins/event_log/server/types';
+
+const delay = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
+
+export default function({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const log = getService('log');
+  const retry = getService('retry');
+
+  describe('Event Log public API', () => {
+    it('should allow querying for events by Saved Object', async () => {
+      const id = uuid.v4();
+
+      const expectedEvents = [fakeEvent(id), fakeEvent(id)];
+
+      await logTestEvent(id, expectedEvents[0]);
+      await logTestEvent(id, expectedEvents[1]);
+
+      await retry.try(async () => {
+        const { body: foundEvents } = await supertest
+          .get(`/api/event_log/event_log_test/${id}/_find`)
+          .set('kbn-xsrf', 'foo')
+          .expect(200);
+
+        expect(foundEvents.length).to.be(2);
```

Comment:
nit: could be moved into a function to remove repetition with the other two `retry.try` below.